packages/ember-runtime/lib/mixins/array.js

```diff
@@ -540,8 +540,23 @@ export default Mixin.create(Enumerable, {
     return an enumerable that maps automatically to the named key on the
     member objects.
 
-    If you merely want to watch for any items being added or removed to the array,
-    use the `[]` property instead of `@each`.
+    @each should only be used in a non-terminal context, and is deprecated when
```

Comment:
This isn't deprecated any more :crying_cat_face:, can you remove mention of the deprecation? (all of other changes look great!)